db/schema.rb

```diff
@@ -10,9 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_05_31_164754) do
-  create_schema "analytics"
-
```

Comment:
how did this happen?

Reply:
I think it was accidentally added by me (Since I had run the analytics.rake locally.)